android/build.gradle.kts

```
@@ -2,6 +2,7 @@ import org.jetbrains.kotlin.konan.properties.loadProperties
 
 plugins {
     id(Plugins.ANDROID_APPLICATION)
+    id(Plugins.KOVER) version Versions.KOVER
```

then we can just add Kover plugin without version

```suggestion
    id(Plugins.KOVER)
```